src/core/lib/slice/slice_internal.h

```diff
@@ -327,4 +327,32 @@ inline grpc_slice grpc_slice_from_static_string_internal(const char* s) {
   return grpc_slice_from_static_buffer_internal(s, strlen(s));
 }
 
+/* Helper method for grpc_slice_buffer_move_into_internal only - used when
+   dst is empty. Only grpc_slice_buffer_move_into_internal() should be used.
+   Think of this as a static method, not part of the internal API. */
+void _helper_grpc_slice_buffer_move_into_empty_buf(grpc_slice_buffer* src,
```

Comment:
A better alternative for these would be to put them in the `grpc_core::slice::internal` namespace, to reflect the fact that they're an internal part of the slice API.